### PR TITLE
Master update numpy

### PR DESCRIPTION
**Pull request by mingwandroid**
@msarahan, @jjhelmus, @TomAugspurger please review.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,19 +12,20 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  script: python setup.py install --single-version-externally-managed --record=record.txt  # [win]
 
 requirements:
   build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: I don't think conda-forge is ready for that yet @mingwandroid

**Comment by msarahan** [comment]
sorry I approved it too quickly.  Wasn't paying attention to which repo the PR was against.

**Comment by mingwandroid** [comment]
Stupid GitHub! This was meant for our fork. Oops.

**Comment by ocefpaf** [comment]
> Stupid GitHub! This was meant for our fork. Oops.

I would love to start getting those updates in conda-forge though :smile:

**Pull request closed without merging**